inc/dbmysql.class.php

```diff
@@ -762,6 +762,9 @@ function __construct ($dbconnexion, $table, $crit="", $debug=false) {
 
       $this->conn = $dbconnexion;
       if (is_string($table) && strpos($table, " ")) {
+//          if ($_SESSION['glpi_use_mode'] == Session::DEBUG_MODE) {
```

Comment:
For now, there is tons of place (especially "old" usage of countElementsInTable, with SQL restriction).
So, IMHO it's too early to raise this warning, which is only for dev, 

But of course, it will have to be enabled later